src/sql/workbench/services/accountManagement/browser/accountManagementService.ts

```diff
@@ -71,6 +72,7 @@ export class AccountManagementService implements IAccountManagementService {
 		this._removeAccountProviderEmitter = new Emitter<azdata.AccountProviderMetadata>();
 		this._updateAccountListEmitter = new Emitter<UpdateAccountListEventParams>();
 		this.configurationService = configurationService;
+		this._authLibrary = this.configurationService.getValue('azure.authenticationLibrary');
```

Comment:
This is still not going to update correctly if the user changes their configuration. 

Although I'm now recalling that you force a restart when the library is changed - is that correct? If that's the case then this is fine (although I would suggest adding a comment explaining that so it's clear).

Reply:
Comment added.